src/rust/src/rdataframe/mod.rs

```diff
@@ -420,7 +422,57 @@ impl DataFrame {
             .map_err(polars_to_rpolars_err)
             .map(DataFrame)
     }
+
+    pub fn write_csv(
+        &mut self,
+        path: Robj,
+        has_header: Robj,
+        separator: Robj,
+        line_terminator: Robj,
+        quote: Robj,
+        batch_size: Robj,
+        datetime_format: Robj,
+        date_format: Robj,
+        time_format: Robj,
+        float_precision: Robj,
+        null_value: Robj,
+        quote_style: Robj,        
+    ) -> RResult<()> {
+
+        let null = robj_to!(Option, String, null_value).unwrap_or_default().unwrap();
```

Comment:
you may want to change this to `robj_to!(string, null_value)?` then NULL is automatically prohibited and return a fine error, and "" is already the default from R.